src/test/java/edu/ksu/canvas/attendance/controller/CourseConfigurationControllerITest.java

```diff
@@ -116,6 +166,7 @@ public void saveCourseConfiguration_MinuteBasedAttendance() throws Exception {
         mockMvc.perform(post("/courseConfiguration/" + irrlevantSectionId + "/save")
                 .param("saveCourseConfiguration", "Save Course Configuration")
                 .param("defaultMinutesPerSession", String.valueOf(expectedDefaultMinutesPerSession))
+                .param("gradingOn", String.valueOf(false))
```

Comment:
You should use a string literal since String.valueOf(false) always returns the same thing.